src/azaryan/Main.java

```diff
@@ -0,0 +1,55 @@
+package azaryan;
+
+import azaryan.annotation.*;
+import azaryan.domain.CustomerDto;
+
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.util.regex.Pattern;
+
+class Main {
```

Comment:
Habet jan I don't see where you validate fields

Reply:
https://github.com/Azaryann/validation/pull/1/commits/db3928c7db64f4460690051f42998c1992a968a0